doc/ref/plotting_options/interactivity.ipynb

```diff
@@ -0,0 +1,297 @@
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(title='Default hover_cols', **opts)\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    hover_cols=['sex', 'body_mass_g'], title='Additional hover columns', **opts\n",
+    ")\n",
+    "plot3 = df.hvplot.scatter(\n",
+    "    hover_cols='all', title='All hover columns', **opts\n",
+    ")\n",
+    "\n",
+    "plot4 = df.hvplot.scatter(\n",
+    "    hover_cols='all',\n",
+    "    use_index=False,\n",
+    "    title='All hover columns without index',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "(plot1 + plot2 + plot3 + plot4).cols(2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2560fd13-ccc7-4be1-805f-08f60912c487",
+   "metadata": {},
+   "source": [
+    "Hover on the points in each of the examples to see the displayed hover information."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e2e2707-5c15-4ae5-bc4d-dc3f14bee454",
+   "metadata": {},
+   "source": [
+    "(options-hover_formatters)=\n",
+    "## `hover_formatters`\n",
+    "\n",
+    "A dictionary that controls how values appear in hover tooltips. Keys are column names or dimensions. Values must be one of:\n",
+    "- 'numeral' – for abbreviated numbers (e.g. 1k, 3.2M)\n",
+    "- 'datetime' – for date/time values\n",
+    "- 'printf' – for precision control like '%.2f'\n",
+    "- CustomJSHover – for custom JavaScript formatting logic\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f41cc94-d57a-4893-a113-55302d045467",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "opts = dict(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    hover_cols='body_mass_g',\n",
+    "    frame_width=250,\n",
+    ")\n",
+    "\n",
+    "plot1 = df.hvplot.scatter(\n",
+    "    hover_tooltips=[('Mass', '@{body_mass_g}{0.0 a}')],\n",
+    "    hover_formatters={'@body_mass_g': 'numeral'},\n",
+    "    title='Using `numeral` formatter',\n",
+    "    **opts\n",
+    ")\n",
+    "plot2 = df.hvplot.scatter(\n",
+    "    hover_tooltips=[('Mass', '@{body_mass_g}{0.2f}')],\n",
+    "    hover_formatters={'@body_mass_g': 'printf'},\n",
+    "    title='Using `printf` formatter',\n",
+    "    **opts\n",
+    ")\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0daabb1e-0ea6-4d81-8f4c-1213d271ee84",
+   "metadata": {},
+   "source": [
+    "::: {seealso}\n",
+    "See [`bokeh formatters`](https://docs.bokeh.org/en/latest/docs/reference/models/formatters.html) for more information about the formatter types.\n",
```

Comment:
Nice to reference to Bokeh's API. If it exists, it'd be nice to link first to a more user-friendly page.